js/common/modules/jsunity.js

```diff
@@ -67,8 +68,8 @@ jsUnity.results.pass = function (index, testName) {
   RESULTS[testName].status = true;
   RESULTS[testName].duration = newtime - STARTTEST;
 
-  print(newtime.toISOString() + internal.COLORS.COLOR_GREEN +  ' [PASSED] ' + testName + internal.COLORS.COLOR_RESET +
-    ' in ' + STARTTEST.toISOString() + '+' + ((newtime - STARTTEST) / 1000).toFixed(3) + 's');
+  print(newtime.toISOString() + internal.COLORS.COLOR_GREEN +  ' [     PASSED ] ' + testName + internal.COLORS.COLOR_RESET +
+    ' (' + RESULTS[testName].duration + ' ms)');
```

Comment:
should we have setup/teardown here too?

Reply:
> I dislike not having the start time anymore - its what gives you the anchor to search for in server logs.

`jsUnity.results.beginSetUp` of `js/common/modules/jsunity.js` has `jsUnity.env.getDate().toISOString()` which indicates a timestamp for a start.

> should we have setup/teardown aded to this time here too?

`STARTTEST` is set at `jsUnity.results.begin` and used in `jsUnity.results.fail` and `jsUnity.results.pass` of `js/common/modules/jsunity.js` so we have it included.